css/at-rules.json

```diff
@@ -120,6 +120,20 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@counter-style"
   },
+  "@container": {
+    "syntax": "@container <container-condition># {\n  <block-contents>\n}",
+    "interfaces": [
+      "CSSGroupingRule",
+      "CSSConditionRule",
+      "CSSContainerRule"
+    ],
```

Comment:
Currently sone at-rule includes its interface and inherited interface, while most are not. This may need desicion, but it OK to not modify this at present.

In docs:

> * `interfaces` (array of strings): These are the [CSSOM](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Object_Model) interfaces that belong to the at-rule.

For this reason I think only `CSSContainerRule` should be represented

Reply:
Mhm, I just followed some of the other at-rules which included the top 2 interfaces as well as their own

Reply:
yes, it is true, so I mean what do you think of this case? If not sure, you could simply keep it.

Reply:
I don't mind, let's keep it for now I guess

Reply:
I think twice, and I think we should not keep inherit interfaces.

i. currently only three cases list inherit interfaces (including one that no list direct interface), while nine cases only list direct interface
ii. the inherit info is a API feature, which is better to maintain at [inheritance.json](https://github.com/mdn/data/blob/main/api/inheritance.json)